hazelcast/src/main/java/com/hazelcast/spi/impl/BasicOperationService.java

```diff
@@ -583,7 +643,12 @@ private void notifyRemoteCall(Response response) {
     /**
      * Responsible for processing an Operation.
      */
-    private final class OperationHandler implements BasicOperationHandler {
+    private final class BasicOperationHandler implements OperationHandler {
```

Comment:
What do you think of ripping this class out of the `BasicOperationService` which is still >1000 lines long? It seems that the `BasicOperationHandler` just uses a few internal members: `node/nodeEngine`, `executedOperationsCount`, `operationBackupHandler` and `backPressureService`. So package private or a constructor shouldn't be that hard to implement.